server/views/partials/layout.njk

```diff
@@ -4,17 +4,18 @@
 {% block head %}
   {% if tagManagerContainerId %}
     <!-- Google Tag Manager -->
-    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    <script nonce="{{ cspNonce }}">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
                 new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
```

Comment:
Interesting - where does the cspNonce come from - is that something that we have generated server side? 
Also is this like the problem you had before where we could provide something in the header for inline scripts?

Reply:
It's created in the setUpWebSecurity middleware file - I added it in my previous PR but I didn't realise I had to add it into the script tags here